### PR TITLE
Adding 4.4.z edges to 4.5 stable channels

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
The nodes are same as 4.5 fast channel.

Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: channels/stable-4.5.yaml [hunk 1]
```diff
@@ -1,6 +1,20 @@
 name: stable-4.5
 versions:
 
+- 4.4.16
```
Comment (sdodson) [inline channels/stable-4.5.yaml:4]: lets drop this for now since #372 hasn't merged yet

Comment (LalatenduMohanty) [reply channels/stable-4.5.yaml]: Fixed it.

**Pull request merged**